aten/src/ATen/function_wrapper.py

```diff
@@ -1991,6 +2103,75 @@ def process_redispatch_cpu_to_hb(option):
         assert alter_method_dispatch
         option['alter_method_dispatch'] = alter_method_dispatch
 
+    def process_fallback_hb_to_cpu(option):
+        # type: (FunctionOption) -> None
+        option['fallback_warning'] = "std::cerr << \"ATen OP {0} falls back to CPU\" << std::endl;\n".format(option['api_name'])
+        fallback_condition = "(c10::probe::use_hb_redispatch() && c10::probe::hb_profiler_is_in_roi() && c10::probe::hb_profiler_is_top_level() && c10::probe::fallback_is_enabled())"
```

Comment:
Here is the fallback condition
 - redispatching is enabled
 - in profiler ROI
 - is top level op
 - fallback is enabled
There are 2 hidden conditons:
 - currently running on HB
 - for general ops, the exception must be HB related.